cdist_manifestation/dependencies.py

```diff
@@ -1,6 +1,16 @@
 from contextlib import contextmanager
 
 _dependencies = []
+_order_dependency = False
+
+
+def order_dependency(value):
```

Comment:
Can't we make this `contextmanager` as well?

```python
with order_dependency():
    typeA()
    typeB()

typeC()
```

Reply:
Yeah, I like that better. Changed now.